.changes/unreleased/Under the Hood-20230425-180020.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Added Support for Testing Table Snapshots
+time: 2023-04-25T18:00:20.57818-07:00
+custom:
+  Author: plypaul
+  Issue: N/A
```

Comment:
Let's just open issues and link them, it's what the tool demands. Also, it's a better place for discussion.

The other option is to add the label to skip changelog - for something like this that might be better, since it doesn't matter much to community members. 

Or you can skip everything in the stack until the last changelog entry.

Reply:
Yeah, skipping sounds good for this set.